tests/String.hs

```diff
@@ -1,3 +1,3 @@
-module String where
+module Strings where
```

Comment:
Was this renamed for a reason? The name doesn't matter to me, but please rename the file if this name is kept.

Reply:
This renamed is to avoid conflict.
String is a built-in class. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String
I change the name of the file.